tests/v2/actions/charts/stackstate.go

```diff
@@ -184,6 +251,16 @@ func newStackstateAgentChartInstallAction(p *payloadOpts, stackstateConfigs *obs
 	return chartInstallAction
 }
 
+func newStackStateChartInstallAction(p *payloadOpts, stackstateConfigs *observability.StackStateConfig, systemProjectID string, additionalValues map[string]interface{}) *types.ChartInstallAction {
```

Comment:
small nit on the function name -- might be more clear to call this `newStackStateServerChartInstallAction` (including `Server`) for clarity, since the other function is already including `Agent`